peerconnection_media_test.go

```diff
@@ -1159,10 +1158,7 @@ func TestPeerConnection_Start_Right_Receiver(t *testing.T) {
 
 	assert.NoError(t, signalPair(pcOffer, pcAnswer))
 
-	pcOffer.negotiationLock.Lock()
-	pcAnswer.negotiationLock.Lock()
-	pcOffer.negotiationLock.Unlock()
-	pcAnswer.negotiationLock.Unlock()
```

Comment:
@jeremija I did this trick of mixing the calls to avoid the linter `empty critical section` error. There should be a better way to fix this.

Reply:
@sgotti thanks - I just disabled the linter for that line using `//nolint`:

```
func waitForNegotiation(pcs ...*PeerConnection) {
	for _, pc := range pcs {
		pc.negotiationLock.Lock()
		pc.negotiationLock.Unlock() //nolint
	}
}
```